src/Reflection/ReflectionParameter.php

```diff
@@ -88,89 +82,46 @@ public function __call($method, $args)
         throw new Exception\BadMethodCallException('Invalid reflection method');
     }
 
-    /**
-     * Retrieve parameter type
-     *
-     * @return string
-     */
-    public function getType()
+    public function getType(): string
     {
         return $this->type;
     }
 
-    /**
-     * Set parameter type
-     *
-     * @param string|null $type
-     * @throws Exception\InvalidArgumentException
-     * @return void
-     */
-    public function setType($type)
+    public function setType(?string $type = null): void
```

Comment:
remove default value from setter

also, i am a bit confused, as in constructor we do not allow nulls, so not sure if we should do it here?

Reply:
Removed, accepting only values of type `string`.